src/App.js

```diff
@@ -4,28 +4,34 @@ import Tile from './Tile';
 import DetailTile from './DetailTile';
 import Details from './Details'
 import './App.css';
-import movieData from './movieData'
+import { fetchAllMovies, fetchMovieDetails } from './apiCalls'
 
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movieData: movieData,
-      movieDetails: ""
+      movieData: [],
+      movieDetails: "",
+      error: ""
     }
   }
 
-seeMovieDetails = (id) => {
-  const movieDetails = this.state.movieData.movies.find(movie => movie.id === id)
-  this.setState({ movieDetails: movieDetails });
-}
+  componentDidMount = () => {
+    fetchAllMovies()
```

Comment:
I like that you've extracted your fetch calls somewhere else, which can help make the `componentDidMount` a little more readable.